test/common/test_context.go

```diff
@@ -521,6 +523,11 @@ func (ctx *TestContext) CleanupAdditionalResources() {
 		panic(err)
 	}
 
+	_, err = ctx.SMRepository.Delete(context.TODO(), types.OperationType)
```

Comment:
Maybe you could reuse the `RemoveAllOperations` and `RemoveAllInstances` here, instead of duplicating logic.
Then we could have also `RemoveAllNotifications`